cumulusci/robotframework/tests/salesforce/forms.robot

```diff
@@ -38,6 +38,15 @@ Require Salesforce object
     Run keyword if  not $result
     ...  log  created object  DEBUG
 
+Go to My Email settings
+    [Documentation]
+    ...  Go directly to the My Email Settings page
+
+    &{org}=  Get org info
+    ${url}=  Set variable
```

Comment:
Is it strictly necessary in Robot to create the variable instead of doing the URL combination in the Go To? I haven't programmed robot in a while and may have forgotten some details. It's super-minor but it doesn't seem to make the code much easier to read.